docs/reference/sql/limitations.asciidoc

```diff
@@ -176,6 +176,25 @@ from `docvalue_fields`.
 [[fields-from-docvalues]]
 === Retrieving from `docvalue_fields`
 
-When the number of columns retrieveable from `docvalue_fields` is greater than the configured <<dynamic-index-settings,`index.max_docvalue_fields_search` setting>>
+When the number of columns retrievable from `docvalue_fields` is greater than the configured <<dynamic-index-settings,`index.max_docvalue_fields_search` setting>>
 the query will fail with `IllegalArgumentException: Trying to retrieve too many docvalue_fields` error. Either the mentioned {es}
-setting needs to be adjusted or fewer columns retrieveable from `docvalue_fields` need to be selected.
+setting needs to be adjusted or fewer columns retrievable from `docvalue_fields` need to be selected.
+
+[float]
+[[aggs-in-pivot]]
+=== Aggregations in the <<sql-syntax-pivot, `PIVOT`>> clause
+
+The aggregation expression in <<sql-syntax-pivot, `PIVOT`>> will currently accept only one aggregation. It is thus not possible to obtain multiple aggregations for any one pivoted columns.
```

Comment:
`...to obtain multiple aggregations for any one pivoted columns.` something is missing here?
Also I'm not native english speaker but how about: `Thus, it is not possible...`

Reply:
> `...to obtain multiple aggregations for any one pivoted columns.` something is missing here?

One extra trailing `s` left after rephrasing. Removed it.

> `Thus, it is not possible...`

That'd also be possible, but a matter of taste. I'd thus leave it as is. :-)